docs/RuntimeAttributes.md

```diff
@@ -369,6 +371,14 @@ runtime {
 }
 ```
 
+#### How to Setup
+
+Configure your Google network to use "Private Google Access".  This will allow your VM's to access Google Services including Google Container Registry, as well as Dockerhub.
+
+1. `gcloud compute networks subnets list`, pick the subnet and region you using w/ Cromwell. If multiple, run the next step for each region & subnet you wish to use.
+1. `gcloud compute networks subnets update [SUBNET-NAME] --region [REGION]  --enable-private-ip-google-access`
+
+That's it!  You can now run with `noAddress` runtime attribute and it will work as expected.
```

Comment:
Even in PAPIv2? I thought our v2 backend still made requests out to docker hub for the helper images?

Reply:
I'm told that Google routes to dockerhub for us, note above I put `as well as Dockerhub`.  It is the magic of their Docker relay feature they put in PAPI to reduce us hammering Dockerhub.

that being said it still warrants a test

Reply:
Nit: replace '&' and 'w/' with actual words.